pyproject.toml

```
@@ -12,6 +12,9 @@ authors = [
     { name = "Brendan Doherty", email = "2bndy5@gmail.com" },
     { name = "Peter Shen", email = "xianpeng.shen@gmail.com" },
 ]
+
+requires-python = ">=3.9"
+
```

```suggestion
requires-python = ">=3.9"
```

No need for extra space here